internal/testing/testdata/exampledata/small-legal-cyclonedx.json

```diff
@@ -0,0 +1,190 @@
+{
+  "bomFormat" : "CycloneDX",
+  "specVersion" : "1.4",
+  "serialNumber" : "urn:uuid:0697952e-9848-4785-95bf-f81ff9731682",
+  "version" : 1,
+  "metadata" : {
+    "timestamp" : "2022-11-09T11:14:31Z",
+    "tools" : [
+      {
+        "vendor" : "OWASP Foundation",
+        "name" : "CycloneDX Maven plugin",
+        "version" : "2.7.1",
+        "hashes" : [
+          {
+            "alg" : "SHA3-512",
+            "content" : "72ea0ed8faa3cc4493db96d0223094842e7153890b091ff364040ad3ad89363157fc9d1bd852262124aec83134f0c19aa4fd0fa482031d38a76d74dfd36b7964"
+          }
+        ]
+      }
+    ],
+    "component" : {
+      "group" : "org.acme",
+      "name" : "getting-started",
+      "version" : "1.0.0-SNAPSHOT",
+      "licenses": [
+        {
+          "license": {
+            "name": "GPL-2.0"
```

Comment:
is this not valid? Based on the spec: https://cyclonedx.org/docs/1.5/json/#tab-pane_components_items_licenses_oneOf_i0_items_license_oneOf_i1

it can be either `id` or `name` (one is required)

Reply:
done.